rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
+
+    @person = person
+    person.rental << self
```

Comment:
- **[OPTIONAL]** I would recommend you use `rentals` as it is more explanatory.

Reply:
## Thank you @DammyShittu for reviewing my project :+1: 
And I thank you a lot for your suggestions and recommendations, 
in the following commit, I have tried to fix all found issues in my project

Thank you once again!